src/printer.js

```diff
@@ -11,6 +11,20 @@ const indent = docBuilders.indent;
 const hardline = docBuilders.hardline;
 const softline = docBuilders.softline;
 
+const makeString = util.makeString;
+
+function stringEscape(str) {
```

Comment:
can you just add a comment explaining why we need to do this? 🙏

Reply:
@mgrip I think I will first create the problem in php parser, let's wait answer

Reply:
sounds good to me 👍

Reply:
if you want to push this just with a comment w/ a link to the issue in the parser repo that works for me - that way we dont have to wait for it to get fixed on their end

Reply:
@mgrip I think we will do this if we do not receive an answer at the earliest acceptable time